docs/gitops.md

```diff
@@ -185,6 +172,49 @@ requests:
 Implementation detail:
 (i.e. How provided inputs get mapped to api behind the scenes):
 
-* `GET https://api.github.com/repos/{owner}/{repo}/releases/tags/{release}`
+* `GET https://api.github.com/repos/{repo}/releases/tags/{release}`
 provides release assets in response.assets
-* request to assets.browser_download_url to get file
+* `GET https://api.github.com/repos/{repo}/releases/assets/${asset.id}`
+provides raw file for asset.id,
+Note header `Accept: application/octet-stream` set by default,
+will override Accept header specified by user.
+
+**Gitlab:**
+
+Release asset links in Gitlab must be a URL that
+returns a valid kubernetes configuration file.
+Ex. `https://raw.githubusercontent.com/razee-io/
+RemoteResource/master/kubernetes/RemoteResource/resource.yaml`
+External URLS should not require headers.
+Gitlab links should be from the same host.
```

Comment:
Not sure my suggestion reads quite like I want either. See if you can improve it?
```suggestion
External URLS must not require headers.
The GET for a Gitlab release artifact link from the same host will include the headers specified for the remote resource.
```